src/actions.js

```diff
@@ -0,0 +1,74 @@
+export const SET_INTERVIEWS = 'SET_INTERVIEWS';
+
+function handleResponse(respnse){
+	return respnse.data.json()
+}
+
+export function setInterviews(interviews){
+	return{
+		type: SET_INTERVIEWS,
+		interviews
+	}
+}
```

Comment:
I see you you only action type SET_INTERVIEWS. There is scope of more actions.  Try to find them out and implement it.